Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/network/NetworkStateProvider.kt

```diff
@@ -117,7 +120,14 @@ class NetworkStateProvider @Inject constructor(
         private val isFakeMeteredConnection: Boolean = false
     ) {
         val isMeteredConnection: Boolean
-            get() = isFakeMeteredConnection || !(capabilities?.hasCapability(NET_CAPABILITY_NOT_METERED) ?: false)
+            get() {
+                val unMetered = if (BuildVersionWrap.hasAPILevel(Build.VERSION_CODES.N)) {
```

Comment:
can we use positive var names and move the negation operator from the return statement to the assignment?